src/components/ConfirmationDialog/ConfirmationDialog.scss

```diff
@@ -69,6 +69,102 @@
   letter-spacing: styles.$letter-spacing--small;
 }
 
+.confirmation-dialog__error-message {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  margin: 8px 0px 8px 0px;
+  padding-left: 0;
+  min-height: 28px;
+  visibility: visible;
+  color: styles.$critical-carmine--light;
+}
+.confirmation-dialog__error-message:empty {
+  visibility: hidden;
+}
+.confirmation-dialog__error-message > svg {
+  height: styles.$icon--medium;
+  width: styles.$icon--medium;
+  opacity: 0.8;
+  margin-left: calc(0.5em - (styles.$icon--medium / 2));
+  flex-shrink: 0;
+}
+.confirmation-dialog__error-message > p {
+  margin: 0px;
+  font-size: styles.$text-size--medium;
+  letter-spacing: styles.$letter-spacing--small;
+}
+
+.confirmation-dialog__text {
+  margin: 0px;
+  font-size: styles.$text-size--medium;
+  letter-spacing: styles.$letter-spacing--small;
+  white-space: pre-line;
+}
+
+.confirmation-dialog__checkbox-wrapper {
+  display: block;
+  position: relative;
+  margin: 0;
+  cursor: pointer;
+  font-size: styles.$text-size--medium;
+  letter-spacing: styles.$letter-spacing--small;
+}
+
+.confirmation-dialog__checkbox {
+  opacity: 0;
+  z-index: -1;
+}
```

Comment:
I'm not sure about this part, there's still an invisible checkbox (which is also clickable) right above the custom one